ophyd/signal.py

```diff
@@ -825,7 +828,8 @@ def get(self, *, as_string=None, connection_timeout=1.0, form='time',
             Use numpy array as the return type for array data.
         timeout : float, optional
             maximum time to wait for value to be received.
-            (default = 0.5 + log10(count) seconds)
+            (default = DEFAULT_TIMEOUT seconds. Add 
+            log10(count) seconds for arrays.)
```

Comment:
I think it is clearer to just say `+ log10(count)`.  'scalars' in EPICS are just length 1 arrays that the clients auto-magically un-pack for you.